src/css/components/footer.scss

```diff
@@ -14,3 +14,12 @@
     color: $color-white;
   }
 }
+
+.content footer {
```

Comment:
This is the footer that appears at the bottom of docs content panels. 

![screen shot 2016-10-28 at 12 50 35 pm](https://cloud.githubusercontent.com/assets/11464021/19820224/293876ae-9d0d-11e6-9e73-407d818781db.png)